### PR TITLE
Added contentDescription containing 'button' via formatters for screen readers (EXPOSUREAPP-1542)

### PR DESCRIPTION
**Pull request by Oliver-Zimmerman**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed, except for minor fixes for typos or grammar mistakes in *documentation or code comments*.
* [x] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)
* [ ] Make sure that your PR does not contain changes in text files, therefore the PR must not contain changes in values/strings/* and / or assets/* (see issue #332 for further information)
* [x] Make sure that your PR does not contain compiled sources (already set by the default .gitignore) and / or binary files

## Description
Added contentDescription with relevant text + "button" appended via formatters where relevant for screen readers. This is a requirement for content that does not contain focusable (readable) subcontent that can still be explored by touch. 

Completes Jira task EXPOSUREAPP-1542.

I am not sure if there is a more simple and generic way to handle this instead of individual formatters for each scenario? I am also not 100% sure on this requirement as the content that _can_ be explored is read, followed by the voiced command 'Double tap to activate'. Perhaps this requirement is redundant?

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -84,6 +84,8 @@
     <string name="accessibility_close">"Schließen"</string>
     <!-- XACT: menu description for screen readers -->
     <string name="accessibility_logo">"Corona-Warn-App"</string>
+    <!-- XACT: button description for screen readers to be appended at the end of content without focusable subcontent that are explored by touch -->
+    <string name="accessibility_button">" Button"</string>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-de/strings.xml:88]: please remove this space here and add it in the formatter

File: Corona-Warn-App/src/main/res/values-en/strings.xml [hunk 2]
```diff
@@ -84,6 +84,8 @@
     <string name="accessibility_close">"Close"</string>
     <!-- XACT: menu description for screen readers -->
     <string name="accessibility_logo">"Corona-Warn-App"</string>
+    <!-- XACT: button description for screen readers to be appended at the end of content without focusable subcontent that are explored by touch -->
+    <string name="accessibility_button">" Button"</string>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-en/strings.xml:88]: same

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -84,6 +84,8 @@
     <string name="accessibility_close">"Close"</string>
     <!-- XACT: menu description for screen readers -->
     <string name="accessibility_logo">"Corona-Warn-App"</string>
+    <!-- XACT: button description for screen readers to be appended at the end of content without focusable subcontent that are explored by touch -->
+    <string name="accessibility_button">" Button"</string>
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/strings.xml:88]: same

**Review by harambasicluka — CHANGES_REQUESTED** [review]
Please also provide tests for the two new formatters.

File: Corona-Warn-App/src/main/res/values-de/strings.xml (hunk 1 above)
Comment (Oliver-Zimmerman) [reply Corona-Warn-App/src/main/res/values-de/strings.xml]: To be added in later PR

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**